adit/upload/templates/upload/upload_job_form.html

```diff
@@ -0,0 +1,70 @@
+{% extends "upload/upload_layout.html" %}
+{% load crispy from crispy_forms_tags %}
+{% block content %}
+    <!--
+        In contrast to the documentation we put hx-ext directly on the form
+        as otherwise the submit button values are not in the sent form data.
+        https://github.com/bigskysoftware/htmx/issues/1337
+-->
+    <div x-data="UploadJobForm($el)" x-cloak>
+        <div id="form_page">{% include "upload/upload_job_form_swappable.html" %}</div>
+        <div id="form_rest">
+            <label for="fileselector">Choose a directory</label>
+            <br>
+            <input id="fileselector"
+                   type="file"
+                   class="btn btn-primary"
+                   @change="toggleUploadButtonVisibility()"
+                   accept=".dcm,.DCM"
+                   webkitdirectory
+                   mozdirectory>
```

Comment:
Which browsers are supported? Do we need a hint somewhere what the browser support is?